deploy/olm-catalog/ibm-block-csi-operator-community/1.5.0/ibm-block-csi-operator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,768 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-block-csi-operator.v1.5.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: ibmcom/ibm-block-csi-operator:1.5.0
+    createdAt: "2021-01-24T22:00:00Z"
+    description: "Run IBM block storage CSI driver on OpenShift."
+    repository: https://github.com/IBM/ibm-block-csi-operator
+    support: IBM
+    alm-examples: >-
+      [
```

Comment:
why the `alm` is not in `yaml` format?

Reply:
https://sdk.operatorframework.io/docs/olm-integration/generation/#csv-fields
> metadata.annotations.alm-examples: CR examples, in JSON string literal format